client/js/components/procedure/StatementSegmentsList/StatementSegment.vue

```diff
@@ -737,6 +737,9 @@ export default {
     },
 
     toggleRecommendationModal () {
+      if (this.asyncComponents[0]) {
+        this.activeId = this.asyncComponents[0].options.id
+      }
```

Comment:
I suggest setting the ID in the then from `loadAddonComponents` . Or is it intended to be reset every time the modal gets reopended